engine/src/cubos/engine/plugins/transform.cpp

```diff
@@ -0,0 +1,34 @@
+#include <cubos/engine/plugins/transform.hpp>
+
+#include <cubos/core/settings.hpp>
+
+#include <components/cubos/position.hpp>
+#include <components/cubos/rotation.hpp>
+#include <components/cubos/scale.hpp>
+#include <components/cubos/local_to_world.hpp>
+
+void applyTransform(cubos::core::ecs::Query<cubos::engine::ecs::LocalToWorld&, const cubos::engine::ecs::Position*,
```

Comment:
Make this function static, to prevent linking with other functions with the same name in other files.